### PR TITLE
DetailView captionOptions & contentOptions attributes

### PR DESCRIPTION
**Pull request by githubjeka**
| Q | A |
| --- | --- |
| Is bugfix? | no |
| New feature? | yes |
| Breaks BC? | no |
| Tests pass? | yes |
| Fixed issues | #11844 fixed by the PR |

Later I will update phpDoc & changeLog...

### CODE REVIEW

**Review by samdark — CHANGES_REQUESTED** [review]
Looks OK. Need CHANGELOG and phpdoc.

**Comment by githubjeka** [comment]
Check in

File: framework/CHANGELOG.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ Yii Framework 2 Change Log
 - Enh #11494: `yii.reloadableScripts` now support wildcards with `*` character (silverfire)
 - Enh #11658: Added argument to `yii\grid\ActionColumn::urlCreator` callback, which holds reference to the column instance (klimov-paul)
 - Enh #11804: Added `yii\behaviors\AttributeTypecastBehavior` for maintaining of strict ActiveRecord attribute types (klimov-paul)
+- Enh #11835: Fixed NTEXT in `\yii\db\mssql\QueryBuilder::$typeMap` that is being deprecated in MSSQL (githubjeka)
```
Comment (samdark) [inline framework/CHANGELOG.md]: Why this line is added?

File: framework/CHANGELOG.md [hunk 2]
```diff
@@ -60,6 +60,8 @@ Yii Framework 2 Change Log
...
+- Enh #11844: Added the ability to customize html attributes of label and value tags in `\yii\widgets\DetailView` (githubjeka)
```
Comment (samdark) [inline framework/CHANGELOG.md]: html → HTML

**Review by samdark — CHANGES_REQUESTED** [review]

File: framework/CHANGELOG.md (hunk 1 above)
Comment (githubjeka) [inline framework/CHANGELOG.md]: Strange. After merge ....

Comment (githubjeka) [inline framework/CHANGELOG.md]: Fixed

File: framework/CHANGELOG.md (hunk 2 above)
Comment (githubjeka) [inline framework/CHANGELOG.md]: Fixed

**Comment by SilverFire** [comment]
Merged https://github.com/yiisoft/yii2/commit/987c302883a9dc3fcfa5011df6559a6f99b8ea91

**Comment by githubjeka** [comment]
Bad merged!

**Comment by SilverFire** [comment]
It's git :)

**Comment by githubjeka** [comment]
No wonder it is so called.

**Comment by SilverFire** [comment]
Resolved by 5b7a70c

**Comment by githubjeka** [comment]
Again it's bad

**Comment by SilverFire** [comment]
Thank you, done af32a48

**Pull request closed without merging**